### PR TITLE
Amol and Richard Trial 2 Tribute Page 

### PR DESCRIPTION
**Pull request by charpeamol**
Tribute page on Nelson Mandella

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <li>He studied law at the University of Fort Hare and the University of Witwatersrand before working as a lawyer in Johannesburg..</li>
+            <li> There he became involved in anti-colonial and African nationalist politics, joining the ANC in 1943 and co-founding its Youth League in 1944</li>
+            <li>After the National Party's white-only government established apartheid, a system of racial segregation that privileged whites, Mandela and the ANC committed themselves to its overthrow</li>
+            <li> He was appointed president of the ANC's Transvaal branch, rising to prominence for his involvement in the 1952 Defiance Campaign and the 1955 Congress of the People</li>  
+          </ul>      
+      </div>
+      <a href=https://en.wikipedia.org/wiki/Nelson_Mandela target="_blank" id="tribute-link">Here is more about this incredible human being on his Wikipedia entry.</a>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
```
Comment (charpeamol) [inline index.html:32]: Please Add HTML CSS linter as it is Necessary for every Microverse Project

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # Trial-Repository
-# Microverse Trial Michel and Amol
+# Microverse Trial Amol and Richard
```
Comment (charpeamol) [inline README.md:2]: Please Document Project in a Professional way as it given information about what project does and how to run it for anyone visiting repo

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <li>A Xhosa, Mandela was born into the Thembu royal family in Mvezo, Union of South Africa</li>
+            <li>He studied law at the University of Fort Hare and the University of Witwatersrand before working as a lawyer in Johannesburg..</li>
+            <li> There he became involved in anti-colonial and African nationalist politics, joining the ANC in 1943 and co-founding its Youth League in 1944</li>
+            <li>After the National Party's white-only government established apartheid, a system of racial segregation that privileged whites, Mandela and the ANC committed themselves to its overthrow</li>
+            <li> He was appointed president of the ANC's Transvaal branch, rising to prominence for his involvement in the 1952 Defiance Campaign and the 1955 Congress of the People</li>  
+          </ul>      
+      </div>
```
Comment (charpeamol) [inline index.html:28]: Use Article tag instead of Main Tag It would be semantically good Practise, Please Make Sure to Close article tag

Comment (charpeamol) [inline index.html:28]: Proper indentation Should be Provided to opening and closing tags. as it helps in debugging

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!
+    </h1>
+    <main id="main">
+      <h1 id="title">Nelson Mandela</h1>
```
Comment (charpeamol) [inline index.html:14]: <h1> used to tell what the overall post is about.  change <h1/> tag given to Nelson Mandela to <h2> tag include keywords relating to the main keywords but offer a particular focus.

**Review by charpeamol — COMMENTED** [review]
#Changes required

File: README.md (hunk 2 above)
Comment (danmainah) [reply README.md:2]: - [ ] [OPTIONAL] It also ensures the project information like what it does and how its run are easily available to anyone visiting the repo.

File: index.html (hunk 4 above)
Comment (danmainah) [reply index.html:14]: - [ ] Kindly rewrite this comment, it's not easily understood, you can visit [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) to learn more about semantics and be able to give your observation clearly

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @charpeamol , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Good use of inline comments.

# Required Changes ♻️

- [ ] Kindly ensure you have a review summary, it serves as a summary of what the review is about and direction on further info like inline comments to fix errors or improve code.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by charpeamol — COMMENTED** [review]
# Required Changes  
Heading Tags Semantic Order 
adding linters 
project documentation 
using proper sematic tags for main content

**Review by KayLemba — APPROVED** [review]
# Project Approved 🚀

Hi @charpeamol  🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights 
- You included an explanation, a "why", for each requested change 🎉 
-  You phrased the comments politely, as suggestions instead of orders 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly include emojis and gifs to make your code review more human-friendly 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**